.eslintignore

```diff
@@ -144,4 +144,5 @@ src/Explorer/Notebook/NotebookRenderer/decorators/kbd-shortcuts/index.tsx
 src/Explorer/Notebook/temp/inputs/connected-editors/codemirror.tsx
 src/Explorer/Tree/ResourceTreeAdapter.tsx
 __mocks__/monaco-editor.ts
-src/Explorer/Tree/ResourceTree.tsx
\ No newline at end of file
+src/Explorer/Tree/ResourceTree.tsx
+src/Utils/PriorityBasedExecutionUtils.ts
```

Comment:
Do not add new files to do list; instead fix the issues.

Reply:
The PriorityBasedExecutionUtils.ts has implementation for the Cosmos plugin and the lint issue is raised for the following method for the any type. 

"export const requestPlugin: Cosmos.Plugin<any> = async (requestContext, next) => {"

I cannot change the definition or signature for this method and reason this object has been added here. There is similar file "src/Common/CosmosClient.ts" which has similar method and that has also been added to this file